src/data/data.js

```diff
@@ -44,23 +44,22 @@ const pageData = {
               url:'https://www.instagram.com'
             }
 
-          
+
           ]
         },
-        
+
       },
       {
         component: 'hero',
         title: '¡Hola! Esto es Ser Hábil',
-        text:
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.',
+        text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.',
         images: [
           {
-            src: '',
+            src: 'Niño.jpg',
```

Comment:
all the names of the files must be in lower case

Reply:
also try to avoid characters like "Ñ" for simplicity, you could name the assets in english as the rest of the written communication about the project seems to be in english.